src/sql/src/func.rs

```diff
@@ -3847,6 +3847,9 @@ pub static MZ_CATALOG_BUILTINS: LazyLock<BTreeMap<&'static str, Func>> = LazyLoc
             params!(String) => UnaryFunc::SeahashString(func::SeahashString) => UInt32, oid::FUNC_SEAHASH_STRING_OID;
             params!(Bytes) => UnaryFunc::SeahashBytes(func::SeahashBytes) => UInt32, oid::FUNC_SEAHASH_BYTES_OID;
         },
+        "starts_with" => Scalar {
+            params!(String, String) => BinaryFunc::StartsWith => Bool, 3696;
```

Comment:
Might make sense to define an `oid` constant here!

Reply:
AFAICT we only introduce OID constants for Materialize-specific functions/types; OIDs that come from PostgreSQL are just inlined in this file. 🤷🏽